buildtools/DevConfigGenerator/DevConfigGenerator/DevConfigWriter.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.Json;
+
+namespace DevConfigGenerator;
+
+public class DevConfigWriter
+{
+    public static void Write(DevConfigFile config, string filePath)
+    {
+        using var stream = File.OpenWrite(filePath);
+        using Utf8JsonWriter writer = new Utf8JsonWriter(stream, new JsonWriterOptions { Indented = true });
+
+        writer.WriteStartObject();
+
+        if (config.Core != null)
+        {
+            writer.WriteStartObject("core");
+
+            writer.WriteBoolean("updateMinimum", config.Core.UpdateServiceMinimum);
+            writer.WriteString("type", config.Core.Type);
```

Comment:
Should validate Type being either `minor` or `patch` in the future. Any other value isn't technically supported right now for basic DevConfigs.